packages/twenty-server/src/engine/metadata-modules/remote-server/remote-table/types/table-schema-column.ts

```diff
@@ -0,0 +1,4 @@
+export type TableSchemaColumn = {
```

Comment:
Didn't we say we would use `DistantTableColumn` that already exists?

Reply:
I also think it's better to specify a bit more this type if possible since this one is exported and most likely specific to remote tables?

Reply:
as discussed, used a common PostgresTableSchemaColumn type